Jenkinsfile

```diff
@@ -28,7 +37,9 @@ pipeline {
       }
       steps {
         withDockerRegistry(url: "https://index.docker.io/v1/", credentialsId: 'dockerbuildbot-index.docker.io') {
-          sh 'make push'
+          withCredentials(DOCKER_HUB_CREDS) {
+            sh 'export $DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE=$DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE && bin/linuxkit pkg push -org docker binfmt'
```

Comment:
Line 4 already sets the env variable, so a `sh 'make push'` should work here in line 41 again.